client/src/features/ProjectPageV2/utils/roleUtils.ts

```diff
@@ -38,3 +41,16 @@ export function toNumericRole(role: string): number {
   const r = stringToRoleOrNone(role);
   return ROLE_MAP[r];
 }
+
+/** Return a sorted copy of the members */
+export function sortedMembers(members: ProjectMemberListResponse) {
```

Comment:
Naming:

```suggestion
export function sortMembers(members: ProjectMemberListResponse) {
```

This allows you to write: `const sortedMembers = sortMembers(members);`

Reply:
In JavaScript `sort` usually means in-place https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

I have renamed the function to `toSortedMembers`.